pyproject.toml

```diff
@@ -75,6 +75,10 @@ count = ''
 quiet-level = 3
 ignore-words-list = "socio-economic"  # British english spelling that isn't covered by the inbuilt dictionary
 
+[tool.mypy]
+ignore_missing_imports = true
```

Comment:
why do we want to ignore missing imports?

Reply:
In most cases, pyomo will mark imported libraries that do not use it as a warning/errors.
This leads to a lot of "mypy spam" that only goes away by using comments or by adding unofficial 'stub' libraries.
The first is bothersome, the second would inflate our `dev.txt` file.

Since the amount of false positives is often quite big (it more than quintuples in our case), I often turn it off.

This still allows us to check our own function annotations using mypy, which is the main goal, I think.